src/api/v4/error-handling/error-list.js

```diff
@@ -0,0 +1,293 @@
+      },
+      'time-series-invalid-aggregation': {
+        messageRegex: 'timeSeriesInvalidAggregation',
+        friendlyMessage: 'Time aggregation must be a valid value. Use carto.dataview.timeAggregation.'
+      },
+      'time-series-invalid-offset': {
+        messageRegex: 'timeSeriesInvalidOffset',
+        friendlyMessage: 'Offset must an integer value between -12 and 14.'
+      },
+      'time-series-invalid-uselocaltimezone': {
+        messageRegex: 'timeSeriesInvalidUselocaltimezone',
+        friendlyMessage: 'useLocalTimezone must be a boolean value.'
+      },
+      'histogram-options-required': {
+        messageRegex: 'histogramOptionsRequired',
+        friendlyMessage: 'Options object to create a histogram dataview is required.'
+      },
+      'histogram-invalid-bins': {
+        messageRegex: 'histogramInvalidBins',
+        friendlyMessage: 'Bins must be a positive integer value.'
+      },
+      'formula-options-required': {
+        messageRegex: 'formulaOptionsRequired',
+        friendlyMessage: 'Formula dataview options are not defined.'
+      },
+      'formula-invalid-operation': {
+        messageRegex: 'formulaInvalidOperation',
+        friendlyMessage: 'Operation for formula dataview is not valid. Use carto.operation'
+      },
+      'category-options-required': {
+        messageRegex: 'categoryOptionsRequired',
+        friendlyMessage: 'Category dataview options are not defined.'
+      },
+      'category-limit-required': {
+        messageRegex: 'categoryLimitRequired',
+        friendlyMessage: 'Limit for category dataview is required.'
+      },
+      'category-limit-number': {
+        messageRegex: 'categoryLimitNumber',
+        friendlyMessage: 'Limit for category dataview must be a number.'
+      },
+      'category-limit-positive': {
+        messageRegex: 'categoryLimitPositive',
+        friendlyMessage: 'Limit for category dataview must be greater than 0.'
+      },
+      'category-invalid-operation': {
+        messageRegex: 'categoryInvalidOperation',
+        friendlyMessage: 'Operation for category dataview is not valid. Use carto.operation'
+      },
+      'category-operation-required': {
+        messageRegex: 'categoryOperationRequired',
+        friendlyMessage: 'Operation column for category dataview is required.'
+      },
+      'category-operation-string': {
+        messageRegex: 'categoryOperationString',
+        friendlyMessage: 'Operation column for category dataview must be a string.'
+      },
+      'category-operation-empty': {
+        messageRegex: 'categoryOperationEmpty',
+        friendlyMessage: 'Operation column for category dataview must be not empty.'
+      }
+    },
+    filter: {
+      'invalid-bounds-object': {
+        messageRegex: 'invalidBoundsObject',
+        friendlyMessage: 'Bounds object is not valid. Use a carto.filter.Bounds object'
+      }
+    }
+  }
+};
+
+function buildErrorCode (error, key) {
```

Comment:
use underscore for private functions so we know they are safe to nuke 💥

Reply:
In this case it's not a function of a class. It's not being exposed since it only exports a function. No need to underscore them.